game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyServerProperties.java

```diff
@@ -1,61 +1,51 @@
 package games.strategy.engine.lobby.client.login;
 
-import java.util.Map;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
 import com.google.common.base.Strings;
 
+import lombok.Builder;
+import lombok.Getter;
+
 /**
  * Server properties.
  *
- * <p>
- * Generally there is one lobby server, but that server may move.
- * </p>
+ * <p>Generally there is one lobby server, but that server may move.
  *
- * <p>
- * To keep track of this, we always have a properties file in a constant location that points to the current lobby
- * server.
- * </p>
+ * <p>To keep track of this, we always have a properties file in a constant location that points to
+ * the current lobby server.
  *
- * <p>
- * The properties file may indicate that the server is not available using the ERROR_MESSAGE key.
- * </p>
+ * <p>The properties file may indicate that the server is not available using the ERROR_MESSAGE key.
```

Comment:
I miss the closing paragraph tags

Reply:
I'm happy with the content, it's just that paragraph tags are in the
```html
<p>
Text
<7P>
```
everywhere in the javadocs, so I'd like to have a consistency.

Reply:
I'm using the guava formatter plug-in to avoid any of these considerations; the file is consistent with itself, soon this package will fill up with more classes and it'll be even more consistent.

Beyond that our team convention is to follow guava java format. I'm neither for or against end-tags necessarily, but I am against us spending time on this when we instead there are other branches waiting for this to be merged and more important problems for us to deal with.. I'll open an issue on formatting..

Reply:
Just noting that the Eclipse formatter (and presumably the Eclipse formatter plugin for IntelliJ) will de-wrap these to place the `<p>` tag on its own line.  So whatever "convention" is chosen should be cross-IDE so we're not constantly reformatting ourselves with extra changes.